### DISCUSSION TITLE
MIUI下小白条不沉浸

### DISCUSSION THREAD

**Discussion by HofNature** [xiaoyaocz/flutter_dmzj#135] [General]
![Screenshot_2023-03-21-02-30-14-207_com.xycz.dmzjx.jpg](https://user-images.githubusercontent.com/35188269/226433685-e6dcae03-dc9e-49d8-ba70-721658de4ccb.jpg)
在MIUI下，导航小白条显示为纯黑底色。

**Comment by xiaoyaocz** [comment]
#139
